lib/parser/lexer.rl

```diff
@@ -2478,15 +2598,7 @@ class Parser::Lexer
 
       e_rbrace | e_rparen | e_rbrack
       => {
-        emit_table(PUNCTUATION)
-
-        if @version < 24
-          @cond.lexpop
-          @cmdarg.lexpop
-        else
-          @cond.pop
-          @cmdarg.pop
-        end
+        e_rbrace_rparen_rbrack
```

Comment:
Same here, also `e_` prefix (I think) means "extended", because these `e_{rbrace,rparen,rbrack}` patterns run a side-effect after matching, and so they can't be easily used in other contexts to match `{`/`(`/`[`. So I think it  should be something like `emit_rbrace_rparen_rbrack`

Reply:
Addressed in 9e32e7d.